samples/CS/XAMLBehaviorsSample/DragPositionBehavior.cs

```diff
@@ -13,6 +8,21 @@ namespace XAMLBehaviorsSample
 {
     public class DragPositionBehavior : DependencyObject, IBehavior
     {
+        private UIElement _parent = null;
+        private Point _prevPoint;
+        private int _pointerId = -1;
+
+        private void _initialise(FrameworkElement element)
```

Comment:
Methjods should not start with underscores and should be PascalCased. See (Contributing Guidelines)[https://github.com/Microsoft/XamlBehaviors/wiki/Contributing-Guidelines#naming--casing]